functions/Get-DbaAgentJob.ps1

```diff
@@ -47,9 +47,9 @@ FUNCTION Get-DbaAgentJob {
 			Returns all SQl Agent Job for the local and sql2016 SQL Server instances
 
 		.EXAMPLE
-			Get-DbaAgentJob -SqlInstance localhost -Enabled True
+			Get-DbaAgentJob -SqlInstance localhost -IncludeDisabled
 
```

Comment:
Disregard this one, reading it wrong on the diff view.

However, if you can include the examples for the `-Job` and `-ExcludeJob` would make it complete examples.